code/modules/sound/soundCache.dm

```diff
@@ -317,7 +317,7 @@ var/global/list/soundCache = list(
  	"sound/effects/valve_creak.ogg" = 'sound/effects/valve_creak.ogg',\
  	"sound/effects/warp1.ogg" = 'sound/effects/warp1.ogg',\
  	"sound/effects/warp2.ogg" = 'sound/effects/warp2.ogg',\
-	"sound/effects/welder_ignite.ogg" = 'sound/effects/welder_ignite.ogg',\
+ 	"sound/effects/welder_ignite.ogg" = 'sound/effects/welder_ignite.ogg',\
```

Comment:
Note: building the soundcache was being fucky as it was trying to add a bunch of audio that's been added recently that *wasn't* given any soundcache spots (even though some definitely should have). I had to go through and manually fix some which is why these unrelated lines are popping up like this. If this PR is merged I'm going to just do a quick PR to add all the other audio files to the cache that hadn't been added like they should have in the first place.